packages/env/lib/config/validate-config.js

```diff
@@ -70,6 +70,13 @@ function validateConfig( config, envLocation ) {
 			);
 		}
 	}
+
+	if ( config.phpVersion && config.phpVersion.length !== 3 ) {
```

Comment:
might be worthwhile adding a check like `typeof config.phpVersion === 'string'` as well

Reply:
After adding this I don't think this logic is correct now. In the case of if `phpVersion` was set to the `7.3` instead of `"7.3"` for example I think the validation will pass now, and that's not what we want. I think I need:
```javascript
if ( config.phpVersion && ( typeof config.phpVersion !== 'string' || ( typeof config.phpVersion === 'string' && config.phpVersion.length !== 3 ) ) {
```
I'm going to need to change this and test a bit to be certain.

Reply:
I confirmed that this expanded conditional with the `||` is the proper check to perform.